### ISSUE TITLE
♿️ Accessibility

### ISSUE THREAD

**Issue by nichelicorn** [nichelicorn/nichelicornDev#30]
- [x] Review Lighthouse / color check
* Ran Accessibility check on 27Dec2021
* Score = 92
* These checks highlight opportunities to improve the accessibility of your web app. Only a subset of accessibility issues can be automatically detected so manual testing is also encouraged.
* Names and labels - These are opportunities to improve the semantics of the controls in your application. This may enhance the experience for users of assistive technology, like a screen reader.

- [ ] Document doesn't have a <title> element
* Internationalization and localization - These are opportunities to improve the interpretation of your content by users in different locales.

-[ ] <html> element does not have a [lang] attribute
* If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.
